### ISSUE TITLE
feat(docs): add a UI installation instructions

### ISSUE THREAD

**Issue by erezrokah** [ghost-inspector/netlify-plugin#4]
This is per our [guidelines](https://github.com/netlify/plugins/blob/main/docs/guidelines.md#provide-a-readme).

You can see an example here https://github.com/netlify-labs/netlify-plugin-lighthouse#usage.

This should be in a separate PR ready to be merged once the plugin is approved.

**Comment by erezrokah** [comment]
🎉
